pkg/pipelineascode/pipelineascode.go

```diff
@@ -26,7 +26,7 @@ const (
 	startingPipelineRunText = `Starting Pipelinerun <b>%s</b> in namespace
```

Comment:
@sm43 any reasons for duplication between here and reconciler?

Reply:
nope. probably have missed to reuse
similar to keys we can have at a single place